Source/Private/ASTextNode2.mm

```diff
@@ -488,7 +488,54 @@ - (id)_linkAttributeValueAtPoint:(CGPoint)point
    inAdditionalTruncationMessage:(out BOOL *)inAdditionalTruncationMessageOut
                  forHighlighting:(BOOL)highlighting
 {
-  AS_TEXT_ALERT_UNIMPLEMENTED_FEATURE();
+  ASDN::MutexLocker l(__instanceLock__);
+
+  ASTextContainer *containerCopy = [_textContainer copy];
+  containerCopy.size = self.calculatedSize;
+  ASTextLayout *layout = [ASTextNode2 compatibleLayoutWithContainer:containerCopy text:_attributedText];
```

Comment:
I don't think it's required for us to copy the container here (or below at line 740). Except in extreme edge cases, `_textContainer` will be at the displayed size. Is this just extra safety, or was this necessary to make it work?